web-common/src/features/canvas/layout-util.ts

```diff
@@ -0,0 +1,227 @@
+import { YAMLMap, YAMLSeq } from "yaml";
+import type { CanvasComponentType } from "./components/types";
+import type {
+  V1CanvasItem,
+  V1CanvasRow as APIV1CanvasRow,
+} from "@rilldata/web-common/runtime-client";
+import { getComponentRegistry } from "./components/util";
+
+const initialHeights: Record<CanvasComponentType, number> = {
+  line_chart: 350,
+  bar_chart: 400,
+  area_chart: 400,
+  stacked_bar: 400,
+  stacked_bar_normalized: 400,
+  markdown: 160,
+  kpi: 200,
+  kpi_grid: 200,
+  image: 420,
+  table: 400,
+};
```

Comment:
I am wondering if these values should be inside the component class. We already have `minSize` and `defaultSize` which are not being used.

Having them inside the class would co-locate all component related config at one place. Secondly, in future if we want these values to be derived from the current properties of the component (for ex. min height for a table with just measures would be different to a table with row dimensions) we can expose a method from the component's class.